vividus-extension-selenium/src/main/java/org/vividus/selenium/DesiredCapabilitiesMerger.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright 2019-2021 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.vividus.selenium;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import org.openqa.selenium.remote.DesiredCapabilities;
+
+public final class DesiredCapabilitiesMerger
+{
+    private DesiredCapabilitiesMerger()
+    {
+    }
+
+    @SuppressWarnings("unchecked")
+    public static DesiredCapabilities merge(DesiredCapabilities base, DesiredCapabilities source)
```

Comment:
it's not clear "who wins" in case of conflict

Reply:
added docs :(